### DISCUSSION TITLE
Question about output log

### DISCUSSION THREAD

**Discussion by darrepac** [abraunegg/onedrive#1481] [Q&A] [answered]
Hi

Just wondering what the following mean:
`Processing XX OneDrive items to ensure consistent local state`

XX is sometimes decreasing, sometimes increasing. What does it mean? what does it represent?

thanks

**Comment by abraunegg** [comment, marked as answer]
@darrepac 
> XX is sometimes decreasing, sometimes increasing. What does it mean? what does it represent?

You have ABC number of files on OneDrive. When starting the client, or performing a sync for the first time, or a re-sync, or depending on client configuration, the client is processing the OneDrive items stored online to validate them.

The API sends these in ~200 bundle lots - there is no way to change this.

This message shows what it is doing when processing these bundles of items from OneDrive.

> what does it mean? what does it represent ?

The answer is clear in the output: to ensure consistent local state
